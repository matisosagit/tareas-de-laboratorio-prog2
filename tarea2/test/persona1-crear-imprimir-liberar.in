crearPersona 12345678 Pedro Alfonso 4 6 2000
imprimirCIPersona
imprimirNombreYApellidoPersona
imprimirPersona
imprimirFechaNacimientoPersona
liberarPersona

crearPersona 23429221 Luis Rodriguez 9 2 1998
imprimirCIPersona
imprimirNombreYApellidoPersona
imprimirPersona
imprimirFechaNacimientoPersona
liberarPersona

crearPersona 98766543 Mateo Perez 29 9 1977
imprimirCIPersona
imprimirNombreYApellidoPersona
imprimirPersona
imprimirFechaNacimientoPersona
liberarPersona

crearPersona 23429221 Jose Riquelme 6 11 1986
imprimirCIPersona
imprimirNombreYApellidoPersona
imprimirPersona
imprimirFechaNacimientoPersona
liberarPersona

crearPersona 55549321 Leopoldo Suarez 23 1 2002
imprimirCIPersona
imprimirNombreYApellidoPersona
imprimirPersona
imprimirFechaNacimientoPersona
liberarPersona

crearPersona 44199264 Mirian Lopez 14 7 1980
liberarPersona
Fin
