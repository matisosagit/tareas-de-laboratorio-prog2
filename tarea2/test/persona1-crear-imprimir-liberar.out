1>2>La CI de la persona es: 12345678
3>El nombre de la persona es: Pedro
El apellido de la persona es: Alfonso
4>Persona Pedro Alfonso
CI: 12345678
Fecha de Nacimiento: 4/6/2000
Perros adoptados:
5>La fecha de nacimiento de la persona es: 4/6/2000
6>7>8>La CI de la persona es: 23429221
9>El nombre de la persona es: Luis
El apellido de la persona es: Rodriguez
10>Persona Luis Rodriguez
CI: 23429221
Fecha de Nacimiento: 9/2/1998
Perros adoptados:
11>La fecha de nacimiento de la persona es: 9/2/1998
12>13>14>La CI de la persona es: 98766543
15>El nombre de la persona es: Mateo
El apellido de la persona es: Perez
16>Persona Mateo Perez
CI: 98766543
Fecha de Nacimiento: 29/9/1977
Perros adoptados:
17>La fecha de nacimiento de la persona es: 29/9/1977
18>19>20>La CI de la persona es: 23429221
21>El nombre de la persona es: Jose
El apellido de la persona es: Riquelme
22>Persona Jose Riquelme
CI: 23429221
Fecha de Nacimiento: 6/11/1986
Perros adoptados:
23>La fecha de nacimiento de la persona es: 6/11/1986
24>25>26>La CI de la persona es: 55549321
27>El nombre de la persona es: Leopoldo
El apellido de la persona es: Suarez
28>Persona Leopoldo Suarez
CI: 55549321
Fecha de Nacimiento: 23/1/2002
Perros adoptados:
29>La fecha de nacimiento de la persona es: 23/1/2002
30>31>32>33>Fin.
